code/script.js

```diff
@@ -30,18 +49,177 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
+const reloadBot = () => {
+  //This allows the user to restart the chatbot
+  userReply('Fy fan nej!');
+  setTimeout(() => //This adds a 1 second delay before the bots chat appears.
+  botReply('No Doggos is a sad doggo-less day for us... but maybe you\'ve changed your mind?')      
+  , 1000);
+  
+    inputWrapper.innerHTML = 
+      `<div>
+      <button id="reloadBtn">Restart</button>
+      </div>`
+
+    document.getElementById('reloadBtn').addEventListener('click', () => {
+    location.reload()})
+}
+
+const nextQuestion = (message) => {
+  console.log(indexDoggos);
+  if (indexDoggos === 1){
+    userReply(message)
+    setTimeout(() => dogSize(message), 2000);
+   } else if (indexDoggos === 2) {
+     userReply(message)
+     setTimeout(() => typeOfDog(message), 2000);
+   } else if (indexDoggos === 3) {
+     userReply(message)
+     setTimeout(() => giftWrapDog(message), 1000);
+   } else if (indexDoggos === 4) {
+     console.log()
+     userReply(message)
+     setTimeout(() => nameofDog(message), 2000);           
+} else {  
+    userReply(message);
+    setTimeout(() => goodbye(message), 2000);
+}
+}
```

Comment:
A nice flow control function. 😸 to make it even better you can change your long `if/else` chain to a more elegant `switch`. You can lear it here https://www.w3schools.com/js/js_switch.asp